core/src/test/java/org/wso2/msf4j/TestUtil.java

```diff
@@ -0,0 +1,113 @@
+    }
+
+    public static void testContent(URI baseURI, String path, String content) throws IOException {
+        testContent(baseURI, path, content, HttpMethod.GET);
+    }
+
+    public static void testContent(URI baseURI, String path, String content, String method) throws IOException {
+        HttpURLConnection urlConn = request(baseURI, path, method);
+        assertEquals(200, urlConn.getResponseCode());
+        assertEquals(content, getContent(urlConn));
+        urlConn.disconnect();
+    }
+
+    public static HttpURLConnection request(URI baseURI, String path, String method) throws IOException {
+        return request(baseURI, path, method, false);
+    }
+
+    public static HttpURLConnection request(URI baseURI, String path, String method, boolean keepAlive)
+            throws IOException {
+        URL url = baseURI.resolve(path).toURL();
+        HttpURLConnection urlConn = (HttpURLConnection) url.openConnection();
+        if (method.equals(HttpMethod.POST) || method.equals(HttpMethod.PUT)) {
+            urlConn.setDoOutput(true);
+        }
+        urlConn.setRequestMethod(method);
+        if (!keepAlive) {
+            urlConn.setRequestProperty(HEADER_KEY_CONNECTION, HEADER_VAL_CLOSE);
+        }
+
+        return urlConn;
+    }
+
+    public static HttpURLConnection request(String path, String method, boolean keepAlive, int port)
+            throws IOException {
+        URL url = URI.create(String.format("http://%s:%d", Constants.HOSTNAME, port)).resolve(path).toURL();
+        HttpURLConnection urlConn = (HttpURLConnection) url.openConnection();
+        if (method.equals(HttpMethod.POST) || method.equals(HttpMethod.PUT)) {
+            urlConn.setDoOutput(true);
+        }
+        urlConn.setRequestMethod(method);
+        if (!keepAlive) {
+            urlConn.setRequestProperty(HEADER_KEY_CONNECTION, HEADER_VAL_CLOSE);
+        }
+
+        return urlConn;
+    }
+
+    public static String getContent(HttpURLConnection urlConn) throws IOException {
+        return new String(IOUtils.toByteArray(urlConn.getInputStream()), Charsets.UTF_8);
+    }
+
+    public static void writeContent(HttpURLConnection urlConn, String content) throws IOException {
+        urlConn.getOutputStream().write(content.getBytes(Charsets.UTF_8));
+    }
+
+    public static boolean isStreamEqual(InputStream input1, InputStream input2) throws IOException {
+        if (!(input1 instanceof BufferedInputStream)) {
+            input1 = new BufferedInputStream(input1);
+        }
+        if (!(input2 instanceof BufferedInputStream)) {
+            input2 = new BufferedInputStream(input2);
+        }
+        int ch = input1.read();
+        while (-1 != ch) {
+            int ch2 = input2.read();
+            if (ch != ch2) {
+                return false;
+            }
+            ch = input1.read();
+        }
+        int ch2 = input2.read();
+        return (ch2 == -1);
+    }
+}
```

Comment:
We need a line break at the end of the file.

Reply:
Line break is already there at the end of the file